interop/grpclb_fallback/client.go

```diff
@@ -0,0 +1,224 @@
+// +build !appengine
```

Comment:
This is unnecessary in light of the line below, as we only support appengine for go1.9.

Reply:
removed